zkbreacharbiter_test.go

```diff
@@ -297,9 +340,8 @@ func testZkBreachSpends(t *testing.T, test breachTest) {
 		t.Fatalf("disputeTx fee of %v sat is less than minRelayTxFee of %v sat", disputeTxFee, minTxFee)
 	}
 
-	// Deliver confirmation of sweep if the test expects it.
-	if test.sendFinalConf {
-		notifier.confChannel <- &chainntnfs.TxConfirmation{}
-	}
-
+	// // Deliver confirmation of sweep if the test expects it.
+	// if test.sendFinalConf {
```

Comment:
Same... Better remove if we don't need it anymore.

Reply:
has been uncommented